modules/react/combobox/lib/ComboboxMenuList.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import {commonColors} from '@workday/canvas-kit-react/tokens';
+import {createElemPropsHook, createSubcomponent} from '@workday/canvas-kit-react/common';
+import {useMenuModel} from '@workday/canvas-kit-react/menu';
+import {ListBox, ListBoxProps} from '@workday/canvas-kit-react/collection';
+
+import {useComboboxModel} from './useComboboxModel';
+
+export interface ComboboxMenuListProps<T = any> extends ListBoxProps<T> {}
+
+/**
+ * The `listbox` uses `aria-labelledby` pointing to the input of a combobox. This input should be
+ * labelled by a form field label for proper accessibility. Use {@link FormField} to ensure proper
+ * accessibility.
+ */
+export const useComboboxMenuList = createElemPropsHook(useMenuModel)(model => {
+  return {
+    role: 'listbox',
+    'aria-labelledby': model.state.id,
+    id: `${model.state.id}-list`,
+    flexDirection: model.state.orientation === 'vertical' ? 'column' : 'row',
```

Comment:
```suggestion
    style: { flexDirection: model.state.orientation === 'vertical' ? 'column' : 'row'},
```

Reply:
idk if we also want to do this in elemPropsHook

Reply:
The direction should not change during the lifecycle of the component. It does not need to be a `style`.

Typically all props that need access to the model should be in the hook. I'm also wondering if we want ALL props to be in the hook.

Reply:
hmm fair point, idk we've made that distinction, my assumption is, if the prop needs access to the model, create an elemProps hooks, if it doesn't then it's not part of the hook

Reply:
Typically if the props access the model, it is part of the behavior of the element. This is why we called them "behavior hooks". But some props don't access the model, but are also part of the behavior like the `role`. Some style props also are part of behavior like this orientation.